plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgreSqlDistributedQueries.java

```diff
@@ -85,5 +87,23 @@ protected TestTable createTableWithDefaultColumns()
                         "col_required2 BIGINT NOT NULL)");
     }
 
+    @Override
+    protected Optional<DataMappingTestSetup> filterDataMappingSmokeTestData(DataMappingTestSetup dataMappingTestSetup)
+    {
+        String typeName = dataMappingTestSetup.getTrinoTypeName();
+        if (typeName.equals("time")
+                || typeName.equals("timestamp(3) with time zone")) {
+            return Optional.of(dataMappingTestSetup.asUnsupported());
+        }
+
+        if (typeName.equals("char(1)")
+                || typeName.equals("varchar(1)")) {
+            // TODO https://github.com/trinodb/trino/issues/3645 - Postgres has different collation than Trino
+            return Optional.empty();
```

Comment:
Postgres -> PostgreSQL

 The issue talks about punctuation symbols, but you're testing with only `A, B, a, b`. I think i gave you a wrong link, sorry.